pnpm-workspace.yaml

```diff
@@ -8,7 +8,6 @@ catalog:
   "@types/node": ^22.2.0
   "@types/react": ^18
   "@types/react-dom": ^18
-  "eslint-config-next": 14.2.11
```

Comment:
Could you please sort the lines here?
(I first thought eslint was missing from here)

Reply:
done 🙂